BearlyHumans/squirrelville
# Squishiness on Ball Collisions

- Ball Squishes when colliding with a new object (i.e. usually will not squish when hitting island walls).
- Settings allow changes to animation curves and the effect of different force values.

Test:
- Go in ISLAND or ensure prefab changes are applied
- Ball mode
- Run into a wall - ball should squish based on curve in settings
- Change settings to adjust the curve, the minimum/maximum force, and the difference force makes on the animation's magnitude.

Notes/Bugs:
> Code could be changed to work on every significant change in velocity, but this would have a large performance effect and only be a little bit better.
